src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -3,103 +3,309 @@
+        - name: --connector-name
+          type: string
+          short-summary: Name of the ACI Connector.
+        - name: --os-type
+          type: string
+          short-summary: Install support for deploying ACIs of this operating system type.
+        - name: --service-principal
+          type: string
+          short-summary: Service principal used for authentication to Azure APIs. If not specified, a new service
+                         principal with contributor role is created and cached at {sp_cache} to be used by subsequent
+                         `az aks` commands.
+        - name: --client-secret
+          type: string
+          short-summary: Secret associated with a service principal. This argument is required if `--service-principal`
+                         is specified.
     examples:
-        - name: Install the ACI-Connector to an AKS Cluster.
-          text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector
-        - name: Install the ACI-Connector for Windows to an AKS Cluster.
+        - name: Install the ACI Connector to a managed Kubernetes cluster.
+          text: >-
+            az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup
+            --connector-name MyConnector
+        - name: Install the ACI Connector for Windows to a managed Kubernetes cluster.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --os-type Windows
-        - name: Install the ACI-Connector for Windows and Linux to an AKS Cluster.
+        - name: Install the ACI Connector for Windows and Linux to a managed Kubernetes cluster.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --os-type Both
-        - name: Install the ACI-Connector using specific SPN.
+        - name: Install the ACI Connector using a specific service principal and secret.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --service-principal <SPN_ID> --client-secret <SPN_SECRET>
-        - name: Install the ACI-Connector from a specific custom chart.
+        - name: Install the ACI Connector from a custom Helm chart.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --chart-url <CustomURL>
-"""
+""".format(sp_cache=SERVICE_PRINCIPAL_CACHE)
+
 helps['aks remove-connector'] = """
     type: command
-    short-summary: Undeploy the ACI-Connector from an AKS cluster.
+    short-summary: Remove the ACI Connector from a managed Kubernetes cluster.
+    long-summary: |-
+        Stops the cluster from deploying Azure Container Instances.
+        See https://github.com/Azure/aci-connector-k8s for more details.
```

Comment:
Actually, their implementation of this command is already changing, so that URL will soon be the wrong reference. I think it's safer if I remove the specifics for now.